lib/plan/plan_itinerary_tabs.dart

```diff
@@ -251,7 +270,7 @@ class PlanItineraryTabPagesState extends State<PlanItineraryTabPages>
         color: theme.primaryColor,
         child: InkWell(
           child: Container(
-            padding: EdgeInsets.all(10.0),
+            padding: EdgeInsets.only(left: 12.0, right: 10.0),
```

Comment:
The left padding here is a bit smaller so that cost and summary are visually on the same line